drivers/ioexpander/gpio.c

```diff
@@ -489,7 +496,65 @@ static int gpio_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 
       case GPIOC_SETPINTYPE:
         {
-          ret = dev->gp_ops->go_setpintype(dev, arg);
+          FAR enum gpio_pintype_e pintype = (FAR enum gpio_pintype_e) arg;
```

Comment:
remove FAR

Reply:
FAR is only used for pointer, not the value type like enum.

Reply:
That's... an excellent point. I somehow had in mind that this was a pointer. Sorry, my bad! Changed.